src/Microsoft.OpenApi.Readers/OpenApiStreamReader.cs

```diff
@@ -49,9 +49,13 @@ public OpenApiDocument Read(Stream input, out OpenApiDiagnostic diagnostic)
             {
                 yamlDocument = LoadYamlDocument(input);
             }
+            catch (SemanticErrorException ex) {
+                diagnostic.Errors.Add(new OpenApiError($"#char={ex.Start.Index},{ex.End.Index}", ex.Message));
```

Comment:
>#char [](start = 57, length = 5)

Is this a standard syntax to denote position of character?

Reply:
[https://tools.ietf.org/html/rfc5147](https://tools.ietf.org/html/rfc5147)  Although I have changed it to #line= in the OpenApiReaderException because char broke tests on the CI due to LF/CRLF differences.  I should change these too to just use line I think

---
In reply to: [194831007](https://github.com/Microsoft/OpenAPI.NET/pull/276#discussion_r194831007) [](ancestors = 194831007)